frontend/src/utils/parseCsvFileToCsvString.ts

```diff
@@ -0,0 +1,46 @@
+import Papa from 'papaparse'
+
+export const parseCsvFileToCsvStringWithoutChunking = (
```

Comment:
/nit: I would assume that this function would naturally parse csv files "without chunking", as my base assumption. I was initially confused that it is explicitly specified that we will not chunk here.

Reply:
resolved!